src/net.js

```diff
@@ -1,5 +1,6 @@
 const Big = require('big.js');
```

Comment:
Sao dậy em? Ko dùng mà thêm vào vậy em?

Reply:
Có dùng mà Thầy, net.js mà không dùng thì file nào mới dùng Thầy. Believe me or not 🧛🏿